demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -746,7 +746,7 @@ def build_rn_desc(
         :return
         The release notes description
         """
-
+        text = bytes(text, "utf-8").decode("unicode_escape")
```

Comment:
do we need to convert it to bytes will this work?
text.encode("utf-8").decode("unicode_escape")